pkg/controller/volume/persistentvolume/pv_controller.go

```diff
@@ -261,6 +264,13 @@ func (ctrl *PersistentVolumeController) syncClaim(ctx context.Context, claim *v1
 	}
 	claim = newClaim
 
+	if utilfeature.DefaultMutableFeatureGate.Enabled(features.VolumeAttributesClass) {
+		_, err := ctrl.assignDefaultVolumeAttributesClass(ctx, claim)
```

Comment:
We should only set a default class if the PVC is unbound.

Reply:
Why the default vac don't take effect on bound pvc?

Reply:
Bound pvc already has some settings. Adding a VAC to a bound PVC should trigger a controller modify volume call instead?

Reply:
When thinking about the upgrade and feature rollout scenario, when we introduce a new default VAC, that's going to opt-in all exisitng PVCs into the feature and start modifying them in bulk. I think that is a risky change for a new feature. I think it is better to make VAC a new opt-in feature.